test/acceptance/helpers/proxy.js

```diff
@@ -50,7 +50,7 @@ export function createApiProxy() {
   app.all('*', (req, res, next) => {
     let path = req.path.replace(/\/$/, '')
     requests[path] = requests[path] || []
-    requests[path].push(req)
+    requests[path].push({ requestDate: new Date(), ...req })
```

Comment:
These `Request` objects already live in a `requests` map, it's probably redundant to call `request.requestDate`. Probably also doesn't need to be a full Date object. What do you think of `{ timestamp: Date.now(), ...req }`?

Reply:
Love it :shipit: